Codeinwp/visualizer
# Add Skip button for second step of Chart Creation

### Summary


Add the possibility to close the second step and the Chart Creation workflow

### Will affect visual aspect of the product

YES

### Screenshots 

![image](https://github.com/Codeinwp/visualizer/assets/17597852/869f5808-8b68-444d-b385-7a54dac4a2a1)


### Test instructions


- Add a New chart and select its type
- You should see the Skip button and when pressed it should close the popup/iframe

## Check before Pull Request is ready:



Closes https://github.com/Codeinwp/visualizer-pro/issues/457
